app/views/candidate_interface/application_form/complete.html.erb

```diff
@@ -14,6 +14,14 @@
   Application submitted on <%= submitted_at_date %>. <%= govuk_link_to 'View application', candidate_interface_application_review_submitted_path %>
 </p>
 
+<% if ProcessState.new(@application_form).state == :ended_without_success %>
```

Comment:
We'll need to come up with something that doesn't use `ProcessState`, because I don't really trust that code yet.

Reply:
It does the right thing for now but it's easy to see it getting out of sync with `ApplicationStateChange`. 

What would a better thing look like? Would it help if we had a list of `UNSUCCESSFUL_END_STATES` on the `ApplicationStateChange`? That way we could start to remove the references to specific statuses from `ProcessState`. Maybe also `SUCCESSFUL_END_STATES` and `UNFINISHED_STATES`.

Reply:
Yeah, I think that would be good to introduce. I'd prefer to duplicate some logic from `ProcessState` for now.